### PR TITLE
Add Ravnica Remastered draft

### PR DESCRIPTION
**Pull request by pfirpfel**
Attempted to recreate the [booster collation ](https://magic.wizards.com/en/news/feature/collecting-ravnica-remastered)as good as possible:

> - 8–10 Commons
> - 3–4 Uncommons
> - 1 Rare or mythic rare
> - 1 Mana slot card
>   - 58% Guildgate (common)
>   - 33% Signet (uncommon)
>   - 9% Chromatic Lantern or a shock land (rare)
> - 1 Retro frame card of any rarity

I ignored the retro slot for now.

### CODE REVIEW

File: forge-gui/res/draft/rankings/rvr.rnk [hunk 1]
```diff
@@ -0,0 +1,405 @@
+#1|Liliana, Dreadhorde General|M|RVR
+#2|Aurelia, Exemplar of Justice|M|RVR
+#3|Aurelia, Exemplar of Justice|M|RVR
```
Comment (paulsnoops) [inline forge-gui/res/draft/rankings/rvr.rnk:3]: Lots of duplicate cards here. There should only be 291 lines. Where did you get the rankings from?

File: forge-gui/res/editions/Ravnica Remastered.txt [hunk 2]
```diff
@@ -3,6 +3,7 @@ Code=RVR
 Date=2024-01-12
 Name=Ravnica Remastered
 Type=Reprint
+Booster=10 Common:!fromSheet("RVR Mana Fixing"):fromsheet("RVR cards"), 3 Uncommon:!fromSheet("RVR Mana Fixing"):fromSheet("RVR cards"), 1 RareMythic:fromSheet("RVR cards"), 1 fromSheet("RVR Mana Fixing")
```
Comment (paulsnoops) [inline forge-gui/res/editions/Ravnica Remastered.txt:6]: Would be preferable to include the retro cards too but not a massive issue, up to you

File: forge-gui/res/blockdata/blocks.txt [hunk 3]
```diff
@@ -128,4 +128,5 @@ The Lord of the Rings: Tales of Middle-earth Jumpstart, -/2/LTR, Meta-Choose(S(L
 Wilds of Eldraine, 3/6/WOE, WOE
 Alchemy: Eldraine,3/6/WOE, YWOE
 The Lost Caverns of Ixalan, 3/6/LCI, LCI
-Alchemy: Ixalan,3/6/LCI, YLCI
\ No newline at end of file
+Alchemy: Ixalan,3/6/LCI, YLCI
+Ravnica Remastered,3/6/RVR, RVR
```
[the remark this answers is not in the document]
Comment (pfirpfel) [reply forge-gui/res/blockdata/blocks.txt:132]: I kept the end of line situation like it was before.

File: forge-gui/res/draft/rankings/rvr.rnk (hunk 1 above)
Comment (pfirpfel) [reply forge-gui/res/draft/rankings/rvr.rnk:3]: It's the exact output of the draft ranking python script in the tools section. The duplicates don't affect the drafting ai. I think it uses the first it finds. My test drafts worked fine.

Comment (paulsnoops) [reply forge-gui/res/draft/rankings/rvr.rnk:3]: I think the script was updated recently so maybe something needs tweaking there from our end. I'm sure it does work fine but I'd still prefer there be no duplicates

Comment (paulsnoops) [reply forge-gui/res/draft/rankings/rvr.rnk:3]: I just tested the old script and it gives the same results so it's the page we're pulling from with the issue. If you don't mind I'll push a commit with the correct list

File: forge-gui/res/editions/Ravnica Remastered.txt (hunk 2 above)
Comment (pfirpfel) [reply forge-gui/res/editions/Ravnica Remastered.txt:6]: It's tricky to implement it properly. They change their approach on collation more or less every (special) set in at least some subtle ways.

Regarding retro cards:
- there is one guaranteed retro card in every draft booster
- if the retro card is rare or mythic, it replaces the regular rare/mythic
- if the retro card is uc/c, it replaces a common
- from my experience opening paper draft boosters, it does not affect the mana fixing slot
- the foil slot (33% of all boosters) replaces a common and can be retro or regular
- not all retro cards appear in draft boosters

Also, there is a small chance of getting a borderless anime card in the rare/mythic slot.

Comment (paulsnoops) [reply forge-gui/res/editions/Ravnica Remastered.txt:6]: Ahh the retro cards are just the frame variation of cards already in the booster so I think it's fine to exclude them. I think this PR is good to go! Thanks!

**Review by paulsnoops — APPROVED** [review]

**Pull request merged**